### PR TITLE
deprecate example

### PR DESCRIPTION
**Pull request by paul-szczepanek-arm**
The netsocket example now includes the extra wifi elements so to cut down on duplication and maintenance this is being removed. Message links to the https://github.com/ARMmbed/mbed-os-example-sockets which is taking over.

### CODE REVIEW

**Comment by paul-szczepanek-arm** [comment]
Wifi is now part of the sockets example

**Review by pan- — APPROVED** [review]
Looks good, have you checked in the documentation and website for references to this example ? 

I just found a reference at https://os.mbed.com/teams/ST/wiki/How-to-make-wifi-tests . 

@donatieng Are you happy with this deprecation notice or do you expect something else ? The netsocket example covers all aspects of this example now.

**Comment by 0xc0170** [comment]
> @donatieng Are you happy with this deprecation notice or do you expect something else ? The netsocket example covers all aspects of this example now.

I'll create a story to document how to deprecate Mbed OS component, we should have this available and followed.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-![](./resources/official_armmbed_example_badge.png)
+## this example has been deprecated
```
Comment (donatieng) [inline README.md:1]: This needs capitalisation

**Review by donatieng — CHANGES_REQUESTED** [review]
@paul-szczepanek-arm Could you elaborate on the reason for deprecation, and maybe add a mention that the Socket examples have WiFi support? Plus maybe a warning icon? @0xc0170 would be worth using this example as a reference for deprecation.

**Review by donatieng — APPROVED** [review]
Thanks Paul, LGTM
@0xc0170 are you happy with this as template?

**Pull request merged**